src/frontend/js/components/CourseGlimpse/index.tsx

```diff
@@ -51,7 +51,9 @@ const CourseGlimpseBase = ({ context, course }: CourseGlimpseProps & CommonDataP
         </div>
       ) : null}
       <div className="course-glimpse__wrapper">
-        <p className="course-glimpse__title">{course.title}</p>
+        <h6 className="course-glimpse__title" title={course.title}>
+          {course.title}
```

Comment:
Why add title? Is it good to repeat?

Reply:
In this way, if user hovers the text, a popup appears and show the full content.

<img width="287" alt="Capture d’écran 2022-01-07 à 17 22 24" src="https://user-images.githubusercontent.com/9265241/148573894-b6d0db8a-e9c1-4155-b920-d16dcb70ec74.png">

Reply:
OK. I thought it would do the same just with the ellusion.